.ocp-indent

```diff
@@ -0,0 +1,2 @@
+match_clause=4
```

Comment:
I don't think we should have both `OCaml` and reason in the repo (at least if there's no really good reason)